app/serializers/serializers/habitat_cover_serializer.rb

```diff
@@ -12,59 +12,76 @@ def serialize
 
       habitat_global_change = Habitat.find_by(name: habitat.to_s).global_change_stat
 
-      stats = reformat(habitat_global_change)
+      sanitised_stats = reformat(habitat_global_change)
 
       habitat_cover_item[:text] =
       I18n.t(
         "home.habitat_cover.habitats.#{habitat}.text",
-        change_percentage: stats[:change_percentage],
-        baseline_year: stats[:baseline_year],
-        recent_year: stats[:recent_year],
-      )
-
-      citation = I18n.t("home.habitat_cover.habitats.#{habitat}.citation",
-        citation: stats[:citations].nil? ? stats[:citation] : stats[:citations][:citation]
+        change_percentage: sanitised_stats[:change_percentage],
+        baseline_year: sanitised_stats[:baseline_year],
+        recent_year: sanitised_stats[:recent_year],
         )
 
-      if stats[:citations]
-        if stats[:citations][:citation_url].nil?
-          citation_url = I18n.t("home.habitat_cover.no_url")
-        else
-          citation_url = I18n.t("home.habitat_cover.habitats.#{habitat}.citation_url",
-            citation_url: stats[:citations][:citation_url]
-            )
-        end
+      citation = sanitised_stats[:citations]
+  
+      unless habitat_global_change.nil? 
+        citation = add_links(habitat_cover_item, sanitised_stats[:citations])
       end
 
-      habitat_cover_item[:change_percentage] = stats[:change_percentage]
+      habitat_cover_item[:modal_content] = citation 
+      habitat_cover_item[:change_percentage] = sanitised_stats[:change_percentage]
       habitat_cover_item[:change_direction] = I18n.t("global.loss_of")
+
       habitat_cover_item[:id] = habitat
-      habitat_cover_item[:modal_content] = citation + "<p>#{citation_url}</p>"
 
       habitat_cover_array.push(habitat_cover_item)
     end
 
     habitat_cover_array
   end
 
+  def add_links(item, citations)
+    arr = []
+
+    citations.each do |citation|
+      
+      citation_text = citation[:citation]
+      if citation[:citation_url].nil?
+        citation_url = I18n.t("home.habitat_cover.no_url")
+      else
+        link = citation[:citation_url]
+        citation_url = "<a target='_' class='modal__link' href='#{link}'>Source</a>"
+      end
+    
+      arr << citation_text + ' ' + citation_url
+    end
+
+    # Newline characters not properly recognised when converted to JSON
+    item[:modal_content] = "<h3>Sources</h3>" + arr.join("<br><br>")
```

Comment:
Can the citations each be a `<p>` element? Ideally a list, but that would be a bigger change. We try to avoid using <br> where possible and use margins for spacing.

Reply:
Yep, I'm going to rewrite this now.